static/js/common.js

```diff
@@ -20,16 +20,16 @@ function password_quality(password, bar, password_field) {
         return undefined;
     }
 
-    var min_length = 6;
+    var min_password_length = 1;
```

Comment:
new thought: https://chat.zulip.org/#narrow/stream/development.20help/subject/password.20length.20checks/near/235231

Let's just leave this as 6 for now in the end (sorry), and then make the change to 0 once Tim is back.